doc/api/tls.md

```diff
@@ -583,6 +583,20 @@ if called on a server socket. The supported types are `'DH'` and `'ECDH'`. The
 
 For Example: `{ type: 'ECDH', name: 'prime256v1', size: 256 }`
 
+### tlsSocket.getFinished()
+
```

Comment:
Please add:

```
 <!-- YAML
 added: REPLACEME
 -->
```

The same for the other function.

Reply:
@BridgeAR Done in the amended commit.